File: PYTHON/pertemuan7/nilai.py
tugas = float(input("Masukan Nilai tugas:"))
uts = float(input("Masukan Nilai UTS:"))
uas = float(input("Masukan NIlai UAS:"))

nilai_akhir=(0.3* tugas)+(0.3* uts)+(0.4* uas)

print ("Nilai Akhir =" ,nilai_akhir)
if nilai_akhir>=60:
    print("Status: LULUS")
else:
    print("Status: TIDAK LULUS")

File: C++/pertemuan7/nilai.cpp
#include <iostream>
using namespace std;

int main()
{
    float tugas,uts,uas,nilaiakhir;

    cout<<"masukan nilai Tugas";
    cin>>tugas;
    cout<<"masukan nilai UTS";
    cin>>uts;
    cout<<"masukan nilai UAS";
    cin>>uas;

    nilaiakhir=(0.3*tugas)+(0.3*uts)+(0.4*uas);

    cout<<"nilai akhir:"<<nilaiakhir<<endl;
    if(nilaiakhir>=60)
        cout<<"status:LULUS";
    else
    {
        cout<<"status: TIDAK LULUS";
    }
    system("pause");

return 0;
}